packages/cli/src/models/files/ProjectFile.ts

```diff
@@ -42,12 +41,29 @@ interface ProjectFileData {
   telemetryOptIn?: boolean;
 }
 
+function getProjectPath(file: string, dir: string): [string, string] {
+  if (!dir) dir = file ? getRootFromFilePath(file) : process.cwd();
+  if (!file) file = getFileFromRoot(dir);
+  return [file, dir];
+}
+
+function getRootFromFilePath(file: string): string {
+  const dir = path.dirname(file);
+  return dir === OPEN_ZEPPELIN_FOLDER ? path.dirname(dir) : dir;
```

Comment:
If I'm not mistaken `file` could be an absolute path.

```suggestion
  return path.basename(dir) === OPEN_ZEPPELIN_FOLDER ? path.dirname(dir) : dir;
```